tape/tape.d.ts

```diff
@@ -1,17 +1,23 @@
-// Type definitions for tape v2.12.3
+// Type definitions for tape v4.2.2
 // Project: https://github.com/substack/tape
-// Definitions by: Bart van der Schoor <https://github.com/Bartvds>
+// Definitions by: Bart van der Schoor <https://github.com/Bartvds>, Haoqun Jiang <https://github.com/sodatea>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 /// <reference path="../node/node.d.ts" />
 
 declare module 'tape' {
-	export = tape;
+	export default tape;
```

Comment:
is this right?

https://github.com/substack/tape/blob/master/index.js#L19
tape does not have `exports.default = module.exports;` syntax.
`export default tape;` is not compatible `export = tape;`.